addons/l10n_nl/migrations/3.3/post-migrate_update_taxes.py

```diff
@@ -22,40 +22,41 @@ def migrate(cr, version):
     goods_taxes = env['account.tax'].browse(_get_tax_ids_for_xml_id(cr, 'btw_X0_producten'))
     services_taxes = env['account.tax'].browse(_get_tax_ids_for_xml_id(cr, 'btw_X0_diensten'))
 
-    old_tax_tags = env['account.account.tag']._get_tax_tags('3bl (omzet)', 'nl')
-    if not old_tax_tags:
+    old_3bl_tax_tags = env['account.account.tag']._get_tax_tags('3bl (omzet)', 'nl')
+    old_3b_tax_tags = env['account.account.tag']._get_tax_tags('3b (omzet)', 'nl')
+    if not old_3bl_tax_tags and not old_3b_tax_tags:
```

Comment:
Don't we need to still do something if `old_3bl_tax_tags` is non empty, even if `old_3b_tax_tags` is empty?